### DISCUSSION TITLE
Utf8Json fork discussions

### DISCUSSION THREAD

**Discussion by gordrs** [Cryptisk/Utf8Json#12] [General]
This is a discussion that is a follow on from https://github.com/neuecc/Utf8Json/issues/223 to discuss the fork.

**Comment by DamianSuess** [comment]
Thank you for kickstarting this again!  Can't wait for the NuGet

**Reply by gordrs** [reply]
I'm preparing it for release this weekend, I'm just unsure what to call it or if I should just bump the version number to 1.4.0. I've been testing it the last few days and actually have it running in production powering internal APIs on .NET 5 now. So far there has been zero issues and a considerable performance increase compared to newtonsoft.json that was running there before.

**Reply by vilinski** [reply]
I think 1.4.0 is totally ok. But would be also cool to take some more critical PRs. Still waiting for [#220](https://github.com/neuecc/Utf8Json/pull/220).

**Reply by gordrs** [reply]
@vilinski Thanks for flagging this one to me, I'll test this one out this evening and merge it in before the nuget package is released.

**Reply by gordrs** [reply]
I merged that in and included it with the 1.4.0 release. https://github.com/Cryptisk/Utf8Json/releases/tag/v1.4.0

**Reply by vilinski** [reply]
You rock! But one drawback of other nuget name - packages, depending on `Utf8Json` have no benefits. For F# community it is `Giraffe` and `Utf8Json.FSharpExtensions`.

**Reply by gordrs** [reply]
True, that is a big drawback. Hopefully neuecc notices and we can get all this stuff merged in, I'd be willing to contribute as a maintainer upstream. Failing that, it would be up to those projects to point to this more updated package if upstream isn't going to get updated in the near future.

**Comment by phobos2077** [comment]
I really doubt nuecc will ever get back to work on this project. He has a track record of making an industry-leading tool, supporting it for some time and then suddenly abandoning altogether. This also happened with UniRx. I really hope some good people will make a similar fork of UniRx at some point, it's super useful and popular.

**Reply by vilinski** [reply]
well I'm afraid too, 2 years of inactivity, not a single reply on github or twitter is enought reasons to take a fork with the nuget name alltogether. 
Can we just publish a `Utf8Json` NuGet again?
